drivers/network/dd/b57xx/b57xxhw.h

```diff
@@ -0,0 +1,892 @@
+/*
+ * PROJECT:     ReactOS Broadcom NetXtreme Driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Hardware specific definitions
+ * COPYRIGHT:   Copyright 2021-2022 Scott Maday <coldasdryice1@gmail.com>
+ */
+#pragma once
+
+#define IEEE_802_ADDR_LENGTH 6
+
+#define MAXIMUM_MULTICAST_ADDRESSES 16
```

Comment:
Why not `64`?

Reply:
Upon inspection of the manual, broadcom used 32 as their multicast list size. Though, I'm not an expert with hashing functions and broadcom doesn't make it clear if this is the maximum.